doc/modules/mod_event_pusher_sns.md

```diff
@@ -13,19 +13,97 @@ Full topics for notifications (ARN as defined in [Amazon Resource Names][aws-arn
-* **plugin_module** (atom, default: 'mod_event_pusher_sns_defaults') - Sets a callback module used for creating user's GUID used in notifications (from user's JID) and for defining custom attributes attached to a published SNS message.
-* **muc_host** (string, default: "conference.@HOST@") - Messages from this MUC host will be sent to the set SNS topic for MUCs.
-* **sns_host** (string, default: unset) - URL to the Amazon SNS service. The URL may be in [virtual host form][aws-virtual-host], and for AWS needs to point at a specific regional endpoint. The scheme, port and path specified in the URL will be used to publish notifications via HTTP POST method.
-* **region** (string, default: unset) - The [AWS region][aws-region] to use for requests.
-* **access_key_id** (string, default: unset) - [ID of the access key][aws-keys] to use for authorization.
-* **secret_access_key** (string, default: unset) - [Secret access key][aws-keys] to use for authorization.
-* **account_id** (string, default: unset) - 12 digit number as defined in [AWS Account Identifiers][aws-acct-identifier] to use for creating TopicArn for publishing notifications.
-* **pool_size** (integer, default: 100) - Worker pool size for publishing notifications
-* **publish_retry_count** (integer, default: 2) - Retry count in case of a publish error
-* **publish_retry_time_ms** (integer, default: 50) - Base exponential backoff time (in ms) for publish errors
+#### `modules.mod_event_pusher_sns.presence_updates_topic`
+* **Syntax:** string
+* **Default:** `""`
+* **Example:** `presence_updates_topic = "user_presence_updated"`
+
+Defines Amazon SNS Topic for presence change notifications. Remove this option to disable these notifications.
+
+#### `modules.mod_event_pusher_sns.pm_messages_topic`
+* **Syntax:** string
+* **Default:** `""`
+* **Example:** `pm_messages_topic = "user_message_sent"`
+
+Defines Amazon SNS Topic for private message notifications. Remove this option to disable these notifications.
+
+#### `modules.mod_event_pusher_sns.muc_messages_topic`
+* **Syntax:** string
+* **Default:** `""`
+* **Example:** `muc_messages_topic = "user_messagegroup_sent"`
+
+Defines Amazon SNS Topic for group message notifications. Remove this option to disable these notifications.
+
+#### `modules.mod_event_pusher_sns.plugin_module`
+* **Syntax:** string
+* **Default:** `""`
+* **Example:** `plugin_module = "mod_event_pusher_sns_defaults"`
+
+Sets a callback module used for creating user's GUID used in notifications (from user's JID) and for defining custom attributes attached to a published SNS message.
+
+#### `modules.mod_event_pusher_sns.muc_host`
+* **Syntax:** string
+* **Default:** `"conference.@HOST@"`
+* **Example:** `muc_host = "conference.HOST"`
+
+Messages from this MUC host will be sent to the set SNS topic for MUCs.
+
+#### `modules.mod_event_pusher_sns.sns_host`
```

Comment:
It's good to add that access_key_id, secret_access_key, region, account_id, sns_host options are required.
Module explicitly checks for them and fails if they are undefined.
So, they *don't have any default values*.